packages/api/.env.test

```diff
@@ -28,4 +28,6 @@ SEGMENT_WRITE_KEY='test'
 REMINDER_TASK_HANDLER_URL=http://localhost:4000/svc/reminders/trigger
 PUBSUB_VERIFICATION_TOKEN='123456'
 PUPPETEER_TASK_HANDLER_URL=http://localhost:9090/
-
+ELASTIC_URL=http://localhost:9200
+BOOKMARKLET_VERSION="latest"
```

Comment:
I think we can remove the bookmarklet environment variables, they shouldn't be used anywhere anymore.